.github/scripts/createLicenseDisclaimerPr.js

```diff
@@ -0,0 +1,33 @@
+const { echo, cd, exec, ShellString, env } = require('shelljs')
+
+const branchName = new ShellString(env.BRANCH_NAME)
+
+cd('packages/mobile')
+
+// ensure that we are using ssh
+exec('git remote set-url origin git@github.com:valora-inc/wallet.git')
+
+echo('Create version bump branch from main')
+exec(`git checkout -b ${branchName}`, { fatal: true })
```

Comment:
Can we globally make all shell [invocations fatal](https://github.com/shelljs/shelljs#configfatal)?

```
config.fatal = true; 
```